mlbgame/__init__.py

```diff
@@ -228,6 +228,12 @@ def game_events(game_id):
     return [mlbgame.events.Inning(data[x], x) for x in data]
 
 
+def important_dates(year=datetime.now().year):
```

Comment:
I suggest using `year=None` here and then assigning `year` to `datetime.now().year` in the function body if `None` is passed in.

The way it's currently done the default value for `year` will be calculated at import time.